dev/benchmarks/macrobenchmarks/macos/Runner/Configs/AppInfo.xcconfig

```diff
@@ -0,0 +1,14 @@
+// Application-level settings for the Runner target.
+//
+// This may be replaced with something auto-generated from metadata (e.g., pubspec.yaml) in the
+// future. If not, the values below would default to using the project name when this becomes a
+// 'flutter create' template.
+
+// The application's name. By default this is also the title of the Flutter window.
+PRODUCT_NAME = macrobenchmarks
+
+// The application's bundle identifier
+PRODUCT_BUNDLE_IDENTIFIER = com.example.macrobenchmarks
```

Comment:
Do we use junk bundle ID and copyright in the iOS app? If not, these should be fixed up.

Reply:
Yes - these values match the iOS versions.